packages/send-emails/client/react/src/SendEmails.js

```diff
@@ -0,0 +1,81 @@
+const { useNylas } = require('@nylas/nylas-react');
+const { useState } = require('react');
+const { styles } = require('./styles');
+
+function SendEmails({ userId }) {
+  const nylas = useNylas();
+
+  const [to, setTo] = useState('');
+  const [body, setBody] = useState('');
+
+  const sendEmail = async ({ userId, to, body }) => {
+    try {
+      const url = nylas.serverBaseUrl + '/nylas/send-email';
+
+      const res = await fetch(url, {
+        method: 'POST',
+        headers: {
+          Authorization: userId,
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ to, body }),
+      });
+
+      const data = await res.json();
+
+      console.log(data);
```

Comment:
Good catch, I guess we don't need this since we're logging it on the return too https://github.com/nylas/use-cases/pull/33/files/d73e0e0b560b6370c47b4742161f23e5d4b30ce9#diff-6f8c4c54ac68e9645b8f53d0507d60ef080e20a969702a414d5613afec91937fR43.